lib/generators/rails_template18f/i18n_js/i18n_js_generator.rb

```diff
@@ -12,45 +12,37 @@ class I18nJsGenerator < ::Rails::Generators::Base
           Install and configure i18n-js gem to provide translations to JS code.
 
           By default, will only export translations with keys that match `*.js.*`
+
+          To use, add the following to your js code:
+
+          1. `import { i18n } from './i18n';`
```

Comment:
question (non-blocking): just checking on the relative path—could it need more explanation or should it be clear from context where the file is?

Reply:
Hmmm - I _think_ it's clear? the `i18n.js` file gets put into `app/javascript/` so it's at the base of all of the app's JS files, right next to `application.js`